deploy/operator/apply/01_FabricGatewayCRD.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: fabricgateways.zalando.org
+  name: "{{{ CRD_PLURAL }}}.zalando.org"
```

Comment:
This switch to templating makes it more difficult for non Zalando users

Reply:
Ah you must've missed it, @irishshagua and I discussed it in the team chat this morning. This was the least bad option for running acceptance tests, and our deploy folder depends on CDP atm anyway. 

Also, the readme contains [a guide](https://zalando-incubator.github.io/fabric-gateway/fabric-gateway-getting-started/#install-api-gateway-operator) for installing the gateway operator which this doesn't affect, so it should be the same installation-wise.